__tests__/installer.test.ts

```diff
@@ -32,8 +32,31 @@ describe('installer tests', () => {
     }
   }, 30000);
 
+  it('Aquires multiple versions of dotnet', async () => {
+    const versions = ['2.2.207', '3.1.120'];
+
+    for (const version of versions) {
+      await getDotnet(version);
+    }
+    installer.DotnetCoreInstaller.addToPath();
+    expect(fs.existsSync(path.join(toolDir, 'sdk', '2.2.207'))).toBe(true);
+    expect(fs.existsSync(path.join(toolDir, 'sdk', '3.1.120'))).toBe(true);
+
+    if (IS_WINDOWS) {
+      expect(fs.existsSync(path.join(toolDir, 'dotnet.exe'))).toBe(true);
+    } else {
+      expect(fs.existsSync(path.join(toolDir, 'dotnet'))).toBe(true);
+    }
+
+    expect(process.env.DOTNET_ROOT).toBeDefined;
+    expect(process.env.PATH).toBeDefined;
+    expect(process.env.DOTNET_ROOT).toBe(toolDir);
+    expect(process.env.PATH?.startsWith(toolDir)).toBe(true);
+  }, 600000);
+
   it('Acquires version of dotnet if no matching version is installed', async () => {
     await getDotnet('3.1.201');
+    installer.DotnetCoreInstaller.addToPath();
```

Comment:
Could you please move `installer.DotnetCoreInstaller.addToPath();` to getDotnet function.